index.html

```diff
@@ -158,7 +162,53 @@ <h2>
             readonly attribute unsigned short angle;
             attribute EventHandler onchange;
           };
+        
+          enum OrientationLockType {
+            "any",
+            "natural",
+            "landscape",
+            "portrait",
+            "portrait-primary",
+            "portrait-secondary",
+            "landscape-primary",
+            "landscape-secondary"
+          };
+
+          enum OrientationType {
+            "portrait-primary",
+            "portrait-secondary", 
+            "landscape-primary",
+            "landscape-secondary"
+          };
         </pre>
+      <p data-dfn-for="OrientationLockType">
+        The <dfn>OrientationLockType</dfn> enum represents the screen
+        orientations to which a screen can be locked: the "<dfn>any</dfn>" enum
+        value represents the <a>any</a> orientation, the "<dfn>natural</dfn>"
+        enum represents the <a>natural</a> orientation, the
+        "<dfn>landscape</dfn>" enum represents the <a>landscape</a>
+        orientation, the "<dfn>portrait</dfn>" enum represents the
+        <a>portrait</a> orientation, the "<dfn>portrait-primary</dfn>" enum
+        represents the <a>portrait-primary</a> orientation, the
+        "<dfn>portrait-secondary</dfn>" enum represents the
+        <a>portrait-secondary</a> orientation, the
+        "<dfn>landscape-primary</dfn>" enum represents the
+        <a>landscape-primary</a> orientation and the
```

Comment:
Please use Oxford commas (insert Brexit joke here). 

```suggestion
        <a>landscape-primary</a> orientation, and the
```